<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>add</class>
 <widget class="QDialog" name="add">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>680</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true"> background-color: #f0f0f0; /* Light grey background */
 padding: 10px; /* Padding inside the dialog */
    font-family: Arial, sans-serif; /* Font family */
    font-size: 12pt; /* Font size */</string>
  </property>
  <widget class="QLabel" name="Title">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>10</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-weight: bold;
color: #333</string>
   </property>
   <property name="text">
    <string>Add new task</string>
   </property>
  </widget>
  <widget class="QLabel" name="Title_task">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>70</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #333; /* Dark grey text for labels */
    font-weight: bold; /* Bold text */</string>
   </property>
   <property name="text">
    <string>Title</string>
   </property>
  </widget>
  <widget class="QLabel" name="Description">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #333; /* Dark grey text for labels */
    font-weight: bold; /* Bold text */</string>
   </property>
   <property name="text">
    <string>Description</string>
   </property>
  </widget>
  <widget class="QLabel" name="date">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>290</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-weight: bold;
 color: #333</string>
   </property>
   <property name="text">
    <string>Due date</string>
   </property>
  </widget>
  <widget class="QLabel" name="Priority">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>375</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-weight: bold;
 color: #333</string>
   </property>
   <property name="text">
    <string>Priority</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Text_lineEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>113</y>
     <width>301</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    background-color: #FFFFFF; /* White background */
    color: #333333; /* Dark text color */
    border: 2px solid #007BFF; /* Blue border */
    border-radius: 5px; /* Rounded corners */
    padding: 5px; /* Padding inside the text edit */
    font-size: 12pt; /* Font size */
}

QLineEdit:focus {
    border: 2px solid #0056b3; /* Darker blue border when focused */
    background-color: #f0f8ff; /* Light blue background when focused */
    outline: none; /* Remove default outline */
}</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="description_edit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>291</width>
     <height>70</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTextEdit {
    background-color: #FFFFFF; /* White background */
    color: #333333; /* Dark text color */
    border: 2px solid #007BFF; /* Blue border */
    border-radius: 5px; /* Rounded corners */
    padding: 5px; /* Padding inside the text edit */
    font-size: 12pt; /* Font size */
}

QTextEdit:focus {
    border: 2px solid #0056b3; /* Darker blue border when focused */
}
</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="date_edit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>330</y>
     <width>301</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QDateEdit {
    background-color: #FFFFFF; /* White background */
    color: #333333; /* Dark text color */
    border: 2px solid #007BFF; /* Blue border */
    border-radius: 5px; /* Rounded corners */
    padding: 5px; /* Padding inside the date edit */
    font-size: 12pt; /* Font size */
}

QDateEdit::drop-down {
    background-color: #007BFF; /* Blue background for dropdown button */
    color: white; /* White text for dropdown button */
    border: none; /* No border for dropdown button */
    border-radius: 5px; /* Rounded corners for dropdown button */
    width: 30px; /* Increase width for better visibility */
}

QDateEdit::drop-down:hover {
    background-color: #0056b3; /* Darker blue on hover */
}

QDateEdit::drop-down:pressed {
    background-color: #004080; /* Even darker when pressed */
}

/* Styling for the arrow buttons */
QDateEdit::up-button, QDateEdit::down-button {
    width: 15px; /* Set a width for the arrows */
    height: 15px; /* Set a height for the arrows */
    background-color: #007BFF; /* Blue background for arrows */
    border: none; /* No border */
    border-radius: 5px; /* Rounded corners for arrows */
}

QDateEdit::up-button:hover, QDateEdit::down-button:hover {
    background-color: #0056b3; /* Darker blue on hover */
}

QDateEdit::up-button:pressed, QDateEdit::down-button:pressed {
    background-color: #004080; /* Even darker when pressed */
}

QDateEdit::calendar-popup {
    background-color: #FFFFFF; /* White background for calendar popup */
    border: 1px solid #007BFF; /* Blue border for calendar popup */
}
</string>
   </property>
  </widget>
  <widget class="QComboBox" name="priority_combo">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>420</y>
     <width>301</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox {
    background-color: #FFFFFF; /* White background */
    color: #333333; /* Dark text color */
    border: 2px solid #007BFF; /* Blue border */
    border-radius: 5px; /* Rounded corners */
    padding: 5px; /* Padding inside the combo box */
    font-size: 12pt; /* Font size */
}

QComboBox::drop-down {
    background-color: #007BFF; /* Blue background for dropdown button */
    color: white; /* White text for dropdown button */
    border: none; /* No border for dropdown button */
    border-radius: 5px; /* Rounded corners for dropdown button */
}

QComboBox::drop-down:hover {
    background-color: #0056b3; /* Darker blue on hover */
}

QComboBox QAbstractItemView {
    background-color: #FFFFFF; /* White background for dropdown items */
    color: #333333; /* Dark text color for dropdown items */
    selection-background-color: #007BFF; /* Blue background for selected item */
    selection-color: white; /* White text for selected item */
}
</string>
   </property>
  </widget>
  <widget class="QPushButton" name="add_butt">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>460</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: #007BFF; /* Blue background for buttons */
    color: white; /* White text */
    border: none; /* No border */
    border-radius: 5px; /* Rounded corners */
    padding: 10px; /* Padding inside buttons */
}

QPushButton:hover {
    background-color: #0056b3; /* Darker blue on hover */
}

QPushButton:pressed {
    background-color: #004494; /* Even darker blue when pressed */
}</string>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QPushButton" name="close_butt">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>510</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: #FF4D4D; /* Bright red background */
    color: white; /* White text */
    border: none; /* No border */
    border-radius: 5px; /* Rounded corners */
    padding: 10px; /* Padding inside the button */
    font-size: 12pt; /* Font size */
}

QPushButton:hover {
    background-color: #D13333; /* Darker red on hover */
}

QPushButton:pressed {
    background-color: #A62828; /* Even darker red when pressed */
}
</string>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QLabel" name="error_label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>570</y>
     <width>371</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel#ErrorLabel {
    color: white; /* White text for error label */
    background-color: red; /* Red background for error label */
    padding: 5px; /* Padding inside the error label */
    border-radius: 5px; /* Rounded corners */
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
